### DISCUSSION TITLE
Werkzeug "Hotlink"

### DISCUSSION THREAD

**Discussion by fanie-mak** [e-netze/webgis-community#24] [Ideas]
Hallo :) 

Es gab in WebGIS 4 das Abfragewerkzeug "Hotlink", wo mit einem Klick in die Karte der passende Link zu einem ausgewählten Thema geöffnet wurde. 

Es wäre toll, wenn dieses Werkzeug auch in WebGIS 5 übernommen werden würde. 
Danke und LG

**Comment by gis-eni** [comment]
Hallo,

eine Frage, den ich schnell beantworten kann:
Dieses Werkzeug wird (in nächster Zeit) nicht umgesetzt.

Das Ziel ist eher mit weniger Werkzeugen zu arbeiten. Die Hotlinks werden im Ergebnis angezeigt. Das funktioniert sogar mit den "Satndardwerkzeug". Der Anwender braucht da gar keine Werkzeug auswählen. 

Das Problem ist hier, dass der ungeschulte Anwender nicht verstehen wird, was diese Werkzeug eigentlich ist. Und was passiert mit Themen, die mehr als einen Hotlink im Ergebnis haben...

Das Argument, dass man sich hier einen Klick erspart und direkt den Link aufruft muss ich entkräften: Dafür muss man öfter den Werkzeug wechseln. Außerdem funktioniert ein Hotlink Werkzeug wie in WebGIS 4 auch nur, wenn der Anwender dazu da Thema auswählt, das abgefragt werden soll => auch wieder mehr Klicks und Verwirrung.

Ein Hotlink in der Ergebnisliste ist ein Web üblicher Link. Dart erwartet sich der Anwender, das er auf eine andere Seite weitergeleitet wird. Beim einen Klick in die Karte eher nicht.

LG
